### PR TITLE
Non-rails project workaround

### PR DESCRIPTION
**Pull request by xyzjace**
I have added a simple condition for non-rails projects which allows a property to be set on the RSpec config.

The setting gives rspec_api_blueprint the project's root directory (under the assumption that the spec_helper.rb, or indeed the RSpec config block is located one folder down from the project's root).

tl;dr
Add to your RSpec config block:

```
config.add_setting :current_dir, :default => File.dirname(__FILE__)+'/..'
```

### CODE REVIEW

**Comment by matteodepalo** [comment]
What if we assume that if Rails is not defined we can directly use `File.dirname(__FILE__)+'/..'` as root path? One less config to worry about. Also if there is no dependency on Rails, we can safely replace `spec.add_dependency 'rspec-rails'` with `spec.add_dependency 'rspec'` in the `rspec_api_blueprint.gemspec`

By the way I've updated the master branch so before doing more changes could you please rebase your branch?

**Comment by xyzjace** [comment]
Honestly I didn't want to go changing the gem up too much. I wasn't sure whether you had intended it to only be for Rails projects, or if you wanted to extend support for non-rails projects. 

The only problem with using File.dirname(**FILE**) is when you run that, it picks up the gems directory as opposed to the RSpec spec helpers directory.

It seems that the config should be injected in to the gem to me, thus avoiding too many assumptions, although I do agree that it's a fair assumption to make that the spec_helper is one folder in from the project root directory.

In short: I'm happy to defer to the direction you want to be taking the Gem. We are using it on a non-rails project, so being able to have that support in there would be fantastic.

Will rebase before next pull request.

**Comment by matteodepalo** [comment]
I usually prefer convention over configuration, so it would be great to find a programmatic way to have the api_docs folder one level up the spec_helper. In any case I would be happy to support non-rails projects and change the gem dependency to `rspec` only; after all there are really minimum requirements for this gem to work.

**Comment by xyzjace** [comment]
I have modified the code to use a convention of expanding on the current file path, a la:

```
File.expand_path('.')
```

if Rails.root doesn't exist. It removes the requirement of injecting config from the RSpec config block. I've updated the README.md to reflect that as well (i.e: removed what I put in there originally).

This way you get the root directory of the project, as the convention for running RSpec is to run it from the root directory of the project.

Moving to RSpec as a gem requirement sounds good. I've got a few other modifications going on as well which I might need to put in as a future pull request.

**Comment by matteodepalo** [comment]
Excellent, thank you!
